cmd/ip-masq-agent/ip-masq-agent.go

```diff
@@ -46,8 +48,14 @@ const (
 
 var (
 	// name of nat chain for iptables masquerade rules
-	masqChain                         utiliptables.Chain
-	masqChainFlag                     = flag.String("masq-chain", "IP-MASQ-AGENT", `Name of nat chain for iptables masquerade rules.`)
+	masqChain     utiliptables.Chain
+	masqChainFlag = flag.String("masq-chain", "IP-MASQ-AGENT", `Name of nat chain for iptables masquerade rules.`)
+
+	// create MASQUERADE iptables rules with --to-ports flag
+	toPorts          interval.Intervals
+	toPortsProtocols = []string{"tcp", "udp", "sctp"}
+	toPortsFlag      = flag.String("to-ports", "", fmt.Sprintf(`Masquerade to specified ports only, example: "1024-29999,32768-65535"; applicable to %s protocols.`, strings.Join(toPortsProtocols, ", ")))
```

Comment:
I know we have not been doing a good job on keeping the doc updated - https://github.com/kubernetes-sigs/ip-masq-agent/blob/master/README.md#agent-flags seems outdated now.

Reply:
Let's move doc updates to a separate PR? I see some other changes needed (e.g. random-fully) and don't want to mix it in here, or causing merge conflicts.